src/doc/src/reference/unstable.md

```diff
@@ -403,28 +401,6 @@ library. The default enabled features, at this time, are `backtrace` and
 `panic_unwind`. This flag expects a comma-separated list and, if provided, will
 override the default list of features enabled.
 
-### timings
-* Tracking Issue: [#7405](https://github.com/rust-lang/cargo/issues/7405)
-
-The `timings` feature gives some information about how long each compilation
-takes, and tracks concurrency information over time.
-
-```sh
-cargo +nightly build -Z timings
-```
-
-The `-Ztimings` flag can optionally take a comma-separated list of the
-following values:
-
-- `html` — Saves a file called `cargo-timing.html` to the current directory
-  with a report of the compilation. Files are also saved with a timestamp in
-  the filename if you want to look at older runs.
-- `info` — Displays a message to stdout after each compilation finishes with
-  how long it took.
-- `json` — Emits some JSON information about timing information.
-
-The default if none are specified is `html,info`.
-
 #### Reading the graphs
```

Comment:
"Reading the graphs" would be good to move out of here to somewhere else. I'm not sure where, though.

Reply:
Will do.

Reply:
Done.